build/linux/cdogs-sdl.appdata.xml

```diff
@@ -24,4 +25,9 @@
     <screenshot>https://raw.githubusercontent.com/cxong/cdogs-sdl/master/wiki/images/screenshots/hqx_new.png</screenshot>
     <screenshot>https://raw.githubusercontent.com/cxong/cdogs-sdl/master/wiki/images/screenshots/quick_play.png</screenshot>
   </screenshots>
+  <releases>
+    <release version="0.7-pre1" date="2019-01-03"/>
```

Comment:
```suggestion
```
This was a test release not suitable for use; is it ok to leave this out?

Reply:
Of course, change the releases section to your heart's content, I just included a few.
The flatpak that I made is built against that release though, but it shouldn't be too much problem to change it to an earlier one.